### ISSUE TITLE
Predictive back

### ISSUE THREAD

**Issue by T8RIN** [olshevski/compose-navigation-reimagined#34]
Will it be added?

**Comment by olshevski** [comment]
@T8RIN Sorry for the late response. My hiatus took too long.

Do you know if the predictive search is already supported somewhere in Compose? The last time I've checked it, it was kinda  non-existent in Compose, but it was a while ago.

**Comment by T8RIN** [comment]
@olshevski Yup, it's existing in modal sheets now, and some components have it too
